### PR TITLE
CSVS-132 Documentation was updated

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

File: docs/logic.md [hunk 1]
```diff
@@ -6,34 +6,36 @@ A controller handles the following events if they have "Completed" status.
...
 The processing of VM creation event consists of the following steps: <br />
-  * a) get the appropriate account ID via CloudStackService <br />
-  * b) try to retrieve tokens from zookeeper. Token can have 'RO' or 'RW' policy <br />
-  * c) if the tokens don't exist: <br />
-    * 1) the tokens are created in vault <br />
-    * 2) zookeeper nodes are created for keeping VM tokens <br />
-    * 3) write the vault tokens to zookeeper (in case of an exception, the tokens will be revoked) <br />
-  * d) write the tokens in CloudStack as UserVM tags <br />
+  1. get the appropriate account ID via CloudStackService: <br />
+    1.1. If the account did not retrieve, skip other steps.
```
Comment (placccebo) [inline docs/logic.md]: doesn't exist

File: docs/logic.md [hunk 2]
```diff
@@ -6,34 +6,36 @@ A controller handles the following events if they have "Completed" status.
...
+  2. try to retrieve tokens from zookeeper. Token can have 'RO' or 'RW' policy <br />
+  3. if the tokens don't exist: <br />
```
Comment (placccebo) [inline docs/logic.md]: If the tokens exist go to the next step, otherwise:

File: docs/logic.md [hunk 3]
```diff
@@ -6,34 +6,36 @@ A controller handles the following events if they have "Completed" status.
...
 The processing of account creation event consists of the following steps: <br />
-  * a) check the account existence <br />
-  * b) try to retrieve tokens from zookeeper. Token can have 'RO' or 'RW' policy <br />
-  * c) if the tokens don't exist: <br />
-    * 1) the tokens are created in vault <br />
-    * 2) zookeeper nodes are created for keeping Account tokens <br />
-    * 3) write the vault tokens to zookeeper (in case of an exception, the tokens will be revoked) <br />
-  * d) write the tokens in CloudStack as Account tags <br />
+  1. check the account existence <br />
+    1.1. If the account did not exist, skip other steps.
```
Comment (placccebo) [inline docs/logic.md]: doesn't exist

File: docs/logic.md [hunk 4]
```diff
@@ -6,34 +6,36 @@ A controller handles the following events if they have "Completed" status.
...
+  2. try to retrieve tokens from zookeeper. Token can have 'RO' or 'RW' policy <br />
+  3. if the tokens don't exist: <br />
```
Comment (placccebo) [inline docs/logic.md]: If the tokens exist go to the next step, otherwise:

**Review by placccebo — CHANGES_REQUESTED** [review]

File: docs/logic.md (hunk 1 above)
Comment (MedvedevBW) [reply docs/logic.md]: ok

File: docs/logic.md (hunk 2 above)
Comment (MedvedevBW) [reply docs/logic.md]: ok

File: docs/logic.md (hunk 3 above)
Comment (MedvedevBW) [reply docs/logic.md]: ok

File: docs/logic.md (hunk 4 above)
Comment (MedvedevBW) [reply docs/logic.md]: ok

**Review by placccebo — APPROVED** [review]

**Pull request merged**